app/services/backfill_application_choice_id.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+class BackfillApplicationChoiceId
```

Comment:
The logic in here is quite straight forward, do we need a class? How often will this be re-used? Wondering why it can't go straight into the data migration?

Reply:
@darokel the only reason was to make it easier (possible) to write tests for the logic. I'm also wondering now whether we should make this a worker - will have to see how long it takes on `proddata`